app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/devices/hplus/HPlusSleepRecord.java

```diff
@@ -0,0 +1,86 @@
+package nodomain.freeyourgadget.gadgetbridge.service.devices.hplus;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Calendar;
+
+
+
+public class HPlusSleepRecord {
+    private long bedTimeStart;
```

Comment:
Could you add a comment about the time units? Every device has its own kind of units (like e.g. year - 2000 for the Mi Band 1).